<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_titel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Vociset herunterladen</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbl_sprache">
         <property name="text">
          <string>Sprache:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbox_sprache">
         <item>
          <property name="text">
           <string>Englisch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Franz√∂sisch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spanisch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Italienisch</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="spacer_sprache">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_suche">
         <property name="text">
          <string>Suche:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_suche"/>
       </item>
       <item>
        <widget class="QPushButton" name="cmd_refresh">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>res/icons/refresh_FILL0_wght500_GRAD0_opsz40.svg</normaloff>res/icons/refresh_FILL0_wght500_GRAD0_opsz40.svg</iconset>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tbl_suche"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="spacer_schliessen">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cmd_schliessen">
         <property name="text">
          <string>Schliessen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
